SSZipArchive/minizip/unzip.c

```diff
@@ -362,6 +362,13 @@ static unzFile unzOpenInternal(const void *path, zlib_filefunc64_32_def *pzlib_f
     else
         us.z_filefunc = *pzlib_filefunc64_32_def;
 
+    if (!(us.z_filefunc.zopen32_file || us.z_filefunc.zfile_func64.zopen64_file) ||
```

Comment:
This fix is perhaps not necessary, but it's good practice and silences 5 "FORWARD_NULL" issues. It ensures that at least one of each pair of function pointers is not NULL.